### PR TITLE
ss-428: Update filter on dashboard page find by wallet with wallet

### PR DESCRIPTION
**Pull request by Holovashcenko**

### CODE REVIEW

**Comment by Holovashcenko** [comment]
image

File: frontend/src/bundles/common/pages/dashboard/dashboard.tsx [hunk 1]
```diff
@@ -76,24 +79,105 @@ type ChartBoxProperties = {
...
     title,
     date,
     controls,
+    transactions = [],
 }: ChartBoxProperties): JSX.Element => {
+    const transactionsByCategory: Record<
+        string,
+        { total: number; transactions: TransactionType[] }
+    > = {};
+    for (const transaction of transactions) {
```
Comment (Evcat6) [inline frontend/src/bundles/common/pages/dashboard/dashboard.tsx:96]: we can create for this helper

File: frontend/src/bundles/common/pages/dashboard/dashboard.tsx [hunk 2]
```diff
@@ -126,11 +210,18 @@ const WalletButton: React.FC<WalletButtonProperties> = ({
     );
 };
 
-const tabsDashboard = [
-    { title: 'Days', isActive: true, disabled: false },
-    { title: 'Week', isActive: false, disabled: false },
-    { title: 'Months', isActive: false, disabled: true },
-];
+// const tabsDashboard = [
```
Comment (Evcat6) [inline frontend/src/bundles/common/pages/dashboard/dashboard.tsx:213]: does we need this in future

File: frontend/src/bundles/common/pages/dashboard/dashboard.tsx [hunk 3]
```diff
@@ -163,8 +247,8 @@ const Dashboard: React.FC = () => {
         defaultValues: { name: '', category: '', wallet: '' },
     });
 
-    const [firstTabs, setFirstTabs] = useButtonTabsState(tabsDashboard);
-    const [secondTabs, setSecondTabs] = useButtonTabsState(tabsDashboard);
+    // const [firstTabs, setFirstTabs] = useButtonTabsState(tabsDashboard);
```
Comment (Evcat6) [inline frontend/src/bundles/common/pages/dashboard/dashboard.tsx:250]: same question

File: frontend/src/bundles/common/pages/dashboard/dashboard.tsx [hunk 4]
```diff
@@ -450,12 +606,12 @@ const Dashboard: React.FC = () => {
                                 <ChartBox
                                     title={'Account Balance'}
                                     date={formatRangeGraph(day)}
-                                    controls={
-                                        <ButtonTabs
-                                            tabsData={firstTabs}
-                                            onClick={setFirstTabs}
-                                        />
-                                    }
+                                    // controls={
```
Comment (Evcat6) [inline frontend/src/bundles/common/pages/dashboard/dashboard.tsx:609]: same question

**Review by Evcat6 — CHANGES_REQUESTED** [review]

**Comment by Holovashcenko** [comment]
image
image
image

File: frontend/src/bundles/common/pages/dashboard/dashboard.tsx (hunk 2 above)
Comment (Holovashcenko) [reply frontend/src/bundles/common/pages/dashboard/dashboard.tsx]: removed

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**